autonomous/autonomous.py

```diff
@@ -32,9 +31,17 @@ class WaypointType(Enum):
 @dataclass
 class Movement:
     type: WaypointType
-    trajectory: Trajectory
+    start_direction: Rotation2d
+    end: Pose2d
+    interior: List[Translation2d]
+    config: TrajectoryConfig
```

Comment:
Do we intend on having different configs for any movement?

Reply:
I think its useful to have (e.g. we might find we want to do the second leg of the five ball slower to let it rotate sooner or we might want to have a move-shoot movement👀), I can make it have a default if your worry was the movement constructor being lengthy